src/components/view/view-event-list.tsx

```diff
@@ -605,7 +611,7 @@ const BuiltInApiRow = observer((p: {
         data-event-id={p.exchange.id}
         tabIndex={p.isSelected ? 0 : -1}
 
-        className={p.isMalicious ? 'malicious' : ''}
+        className={p.isSelected ? 'malicious' : '' }
```

Comment:
should be `className={p.isSelected ? 'selected' : '' }`?

Reply:
you are correct